acceptance-tests/features/support/functions.rb

```diff
@@ -1,343 +1,990 @@
-def create_title_in_db(title_data)
+def create_proprietor_title_in_db(title_data)
 
   title_number = title_data[:title_number]
   postcode = title_data[:postcode]
   street_name = title_data[:street_name]
   house_no = title_data[:house_no]
   town = title_data[:town]
+  surname = title_data[:surname]
+  forename = title_data[:forename]
+  name_category = title_data[:name_category]
+  full_text = title_data[:full_text]
+  multi_proprietors = title_data[:multi_proprietors]
+  if multi_proprietors == 'TwoPI'
+    multiple_proprietors_json = ",{
```

Comment:
Maybe, for readability, it would be a good idea to extract JSON values into separate .json resource files?